src/Microsoft.OData.Core/Json/BufferingJsonReader.cs

```diff
@@ -531,9 +531,10 @@ await this.StartBufferingAsync()
 
             try
             {
-                Tuple<bool, ODataError> readInStreamErrorPropertyResult = await this.TryReadInStreamErrorPropertyValueAsync()
+                (bool IsSuccess, ODataError Error) readInStreamErrorPropertyResult = await this.TryReadInStreamErrorPropertyValueAsync()
                     .ConfigureAwait(false);
-                return readInStreamErrorPropertyResult;
+
+                return readInStreamErrorPropertyResult.ToTuple();

```

Comment:
Is this call `ToTuple` necessary here?

Reply:
I changed the `private` method to use value tuples: `(T, U)`. But the internal methods that's that is referenced by other calls returns a `Tuple<T, U>`. `ToTuple` tranforms a `ValueTuple` into a `Tuple`.

We could change the internal methods to return a `ValueType` as well but I was concerned about scope creep.

Reply:
@ElizabethOkerio I have changed the internal methods signature to return a value tuple and removed the `.ToTuple()` method call.